api/v1/serializers/machine_request_serializer.py

```diff
@@ -36,20 +36,20 @@ class MachineRequestSerializer(serializers.ModelSerializer):
                                          source='new_machine_owner',
                                          queryset=AtmosphereUser.objects.all()
     )
-    vis = serializers.CharField(source='new_machine_visibility')
-    version = serializers.CharField(source='new_machine_version',
+    vis = serializers.CharField(source='new_application_visibility')
```

Comment:
This is API v1, doing so would affect backwards compatibility.. However I agree on API v2 name changing!